client/less/challenge.less

```diff
@@ -1,231 +1,243 @@
 .challenges-instructions-panel {
-  clear: both;
-  overflow-x: hidden;
-  overflow-y: auto;
-  padding-left: 5px;
-  padding-right: 5px;
-  margin-bottom: 10px;
+    clear: both;
```

Comment:
I have made the changes as requested. Sorry for inconvenience. I didn't pay attention to the formatting and my text editor messed it all up.